sass/components/_popover.scss

```
@@ -144,6 +144,7 @@
     .popover__row {
         margin: 0 -15px 0;
         padding: 9px 15px 0;
+        border-color: var(--center-channel-color);
```

Just a small tweak to add the opacity from the original changeCss line. :)
```suggestion
        border-color: rgba(var(--center-channel-color-rgb), 0.2);
```

---

Took the liberty of committing the change directly for you since it took me a bit to get to this review, hope that's okay!